### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by yaneski2000** [sherlock-project/sherlock#2163] [Q&A] [answered]
hello everyone , i hope you are all well. when i try to install sherlock i meet errors wich are error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 7482 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
failed to run git: exit status 128

**Comment by ppfeister** [comment]
How are you trying to install Sherlock? What commands are you running?

**Comment by yaneski2000** [comment]
I use git clone repo sherlock-project/sherlock

Le lun. 10 juin 2024, 15:26, Paul Pfeister ***@***.***> a
écrit :

**Comment by ppfeister** [comment, marked as answer]
That's not a valid command, so it's assumed that you used `git clone https://github.com/sherlock-project/sherlock` instead...in which case, that error message appears to be related to your connection with GitHub and **not** related to Sherlock.

Nonetheless, not a currently supported install method. Cloning the repository only fetches the source code. Please see https://github.com/sherlock-project/sherlock?tab=readme-ov-file#installation for supported methods.
